### PR TITLE
Updating More Dependencies 

### PR DESCRIPTION
**Pull request by aleenaloves**
apollo
--> backwards compatible
emotion
--> backwards compatible
storybook
--> needed to be upgraded to webpack5
--> Documentation:
- https://storybook.js.org/docs/react/builders/webpack#webpack-5
- https://storybook.js.org/blog/storybook-for-webpack-5/

_only updated in the `/client` folder_

### CODE REVIEW

**Review by evmiguel — APPROVED** [review]
LGTM and tested!

File: client/package.json [hunk 1]
```diff
@@ -58,13 +60,13 @@
     "@babel/plugin-transform-runtime": "^7.21.0",
     "@babel/preset-env": "^7.8.7",
     "@babel/preset-react": "^7.8.3",
+    "@storybook/addon-a11y": "^6.5.16",
```
Comment (howard-e) [inline client/package.json:63]: Really small nit is that the alphabetical order has changed which could potentially cause a weird conflict considering the dependency updates happening elsewhere.

**Review by howard-e — APPROVED** [review]
LGTM!

File: client/package.json (hunk 1 above)
Comment (aleenaloves) [reply client/package.json:64]: nice catch ! fixed

**Pull request merged**